Add animation to carrousel

antsgar opened the pull request:
**Description**

Added animation to cards carrousel when you swap cards.

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [x] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

SayakaOno reviewed (approved):
The animation looks nice :)

kathyavini commented on packages/webapp/src/components/CardsCarrousel/styles.module.scss:
```diff
@@ -39,22 +39,49 @@
   }
 }
 
+@keyframes card-to-front {
```
This animation is defined but not used.

kathyavini commented on packages/webapp/src/components/CardsCarrousel/styles.module.scss:
```diff
@@ -39,22 +39,49 @@
+.inactiveCardButton {
+  height: 100%;
+  width: calc(100% - 2px);
+  overflow: hidden;
```
For me this cuts off the icons on the left. Is that the intended behaviour?

![Screenshot 2023-11-06 at 11 17 11 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/e326ffc9-ba1c-4bc9-a11c-c36f89de7179)

antsgar replied:
It was intended to cut them off in mobile when the cards are more overlapped, but they shouldn't look like this in desktop! Will take a look

kathyavini reviewed (commented):
It's subtle, but really nice!!

I just had two small questions which I left in-line.

Also (and it's not from this PR so please feel free to direct elsewhere) -- in looking really closely at the icons, I noticed the profit-loss icon is quite pixelated when it's on the active card. It looks like the SVG set might need replacing with something a little cleaner, is that already on your radar?

![Screenshot 2023-11-06 at 11 23 06 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/0ff43549-35f9-4eb7-b787-6871ac3de84b)

antsgar commented:
@kathyavini I addressed your comments! I do see the profit loss icon pixelated, but I think I might have to swap SVGs with PNGs or maybe export only the icon as SVG to fix it. I'll create a ticket and take a look later
